verify/verify.go

```diff
@@ -866,20 +866,52 @@ func verifyHash256(quote *pb.QuoteV4) error {
 	return nil
 }
 
-func isSvnHigherOrEqual(svn []byte, components []pcs.TcbComponent) bool {
-	for i := range svn {
-		if svn[i] < components[i].Svn {
+func isCPUSvnHigherOrEqual(pckCertCPUSvnComponents []byte, sgxTcbcomponents []pcs.TcbComponent) bool {
+	for i := range pckCertCPUSvnComponents {
```

Comment:
It seems this check was missed previously. To prevent any issues, would it be better to verify that sgxTcbComponents.len() == pckCertCPUSvnComponents.len()? WDYT?

Reply:
... and that the length matches the expected size as per spec unless this is already checked while parsing.

Reply:
It's weird that even SGX DCAP library doesn't have size checks. But it's a good precaution so I'll return false instead of error so that the loop continues. Ultimately, the "no matching TCB level found" error should suffice.

Reply:
While parsing we already fetch 16 bytes for TeeTcbSvn in TDQuoteBody